packages/bus-extract-links/lib/ActorExtractLinks.ts

```diff
@@ -15,11 +16,14 @@ import type * as RDF from '@rdfjs/types';
  * @see IActorExtractLinksOutput
  */
 export abstract class ActorExtractLinks extends Actor<IActionExtractLinks, IActorTest, IActorExtractLinksOutput> {
+  protected readonly labelLinksWithReachability: boolean;
+  protected reachabilityLabel: string;
   /**
    * @param args - @defaultNested {<default_bus> a <cc:components/Bus.jsonld#Bus>} bus
    */
   public constructor(args: IActorExtractLinksArgs) {
     super(args);
+    this.labelLinksWithReachability = args.labelLinksWithReachability ?? false;
```

Comment:
Not sure we need this one. I guess we could just use the actor name `this.name`?